content/en/tracing/profiling/_index.md

```diff
@@ -74,10 +74,10 @@ The Datadog Profiler requires [Java Flight Recorder][1]. The Datadog Profiling l
 
 The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python 3.5+. To begin profiling applications:
 
-1. Install `ddtrace` with the `profile` flavor, which contains both tracing and profiling:
+1. Install `ddtrace`:
 
     ```shell
-    pip install ddtrace[profiling]
+    pip install ddtrace
     ```
 
      **Note**: Profiling is available in the `ddtrace` library for versions 0.36+.
```

Comment:
Maybe we should add a note along the lines of

> For versions 0.36-0.xx, `pip install ddtrace[profiling]`. After that it was moved into the main distribution.

with whatever terminology is best. Otherwise I can see some support cases where people were already using tracing 0.36, try to enable profiling, and it fails in a confusing way.

Reply:
Do you really think it's necessary to document older version of the library?

Reply:
I'm not totally sure, just trying to avoid having people trip on this.

Reply:
I expect people reading this being first-timers, no reason they would trip on it.

Reply:
They'll be first timers to profiling, but some will be existing apm users who will have older versions of ddtrace installed. @hugokash @GaurabAryal in case you have thoughts. Non-blocking if anyone's really opposed to this.

Reply:
See https://github.com/DataDog/documentation/pull/7576/files#r432430330